hail/src/test/scala/is/hail/expr/ir/FoldConstantsSuite.scala

```diff
@@ -6,19 +6,6 @@ import org.scalatest.testng.TestNGSuite
 import org.testng.annotations.Test
 
 class FoldConstantsSuite extends TestNGSuite {
-  @Test def test_let_evaluation() {
-    val ir = Let(
-      "x",
-      I32(5),
-      Let(
-        "y",
-        ApplyBinaryPrimOp(Add(), Ref("x", TInt32()), I32(10)),
-        ApplyBinaryPrimOp(Add(), Ref("y", TInt32()), I32(15))
-      )
-    )
```

Comment:
Why did you delete this test?

Reply:
I removed this functionality from FoldConstants because it doesn't belong there (and Tim built it into a separate pass, ForwardLets).